e2e_test/streaming/time_window_filter.slt

```diff
@@ -0,0 +1,42 @@
+statement ok
+create table t1(v1 int, v2 int, v3 int, v4 int, ts timestamp);
+
+statement ok
+insert into t1 values
+    (1, 1, 4, 6,  '2022-01-01 10:00:00'),
+    (2, 3, 3, 9, '2022-01-01 10:05:00'),
+    (3, 2, 2, 6, '2022-01-01 10:14:00'),
+    (4, 1, 1, 6, '2022-01-01 10:22:00'),
+    (5, 3, 5, 8, '2022-01-01 10:33:00'),
+    (6, 2, 7, 0, '2022-01-01 10:42:00'),
+    (7, 1, 6, 8, '2022-01-01 10:51:00'),
+    (8, 3, 8, 1, '2022-01-01 11:02:00');
+
+statement ok
+flush;
+
+statement ok
+create materialized view m1 as
+with cte as (select * from hop(t1, ts, interval '1' minute, interval '3' minute))
+select * from cte
+where v1>0 AND v2>1 AND v3>2 AND ts >= timestamp '2022-01-01 10:00:00'
+AND window_start >= timestamp '2022-01-01 09:00:00' AND window_end <= timestamp '2022-01-01 10:50:00'
+AND window_start <= ts + interval '1' minute AND window_end > ts + interval '1' minute
+ORDER BY window_start ASC, ts ASC;
+
+statement ok
+flush;
+
+query TTTIIII
+select window_start, window_end, ts, v3, v4, v2, v1 from m1 ORDER BY window_start ASC, ts ASC;
```

Comment:
This is currently a workaround for: https://github.com/risingwavelabs/risingwave/issues/6905

Reply:
seems expected, I shall remove the earlier order by in mview.